content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -105,20 +105,28 @@ manually by the cluster admin, either by means of the underlying Linux
 distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DaemonSet
 with privileged containers.
 
-The sysctl feature is an alpha API. Therefore, sysctls are set using annotations
-on pods. They apply to all containers in the same pod.
+The sysctl feature is a {{< feature-state for_k8s_version="v1.11" state="beta" >}} API.
```

Comment:
This is not the correct way to use this shortcode. It isn't used in the flow of a paragraph, but on its own as a block-level element. Suggest just changing this sentence to only be the shortcode itself.

Reply:
Done.